testing/mediainfo/APKBUILD

```diff
@@ -0,0 +1,31 @@
+# Contributor:
+# Maintainer:
+pkgname="mediainfo"
+pkgver="0.7.89"
+pkgrel="0"
+pkgdesc="Unified display of the most relevant technical and tag data for video and audio files."
+url="https://mediaarea.net/"
+arch="all"
+license="Simplified BSD"
+depends=""
+makedepends=""
+install=""
+subpackages=""
+source="https://mediaarea.net/download/binary/mediainfo/$pkgver/MediaInfo_CLI_$pkgver_GNU_FromSource.tar.xz"
```

Comment:
There’s no such variable `$pkgver_GNU_FromSource`… You must wrap it in braces in this case: `MediaInfo_CLI_${pkgver}_GNU_FromSource`.

BTW It’s really not needed nor desirable to write checksums by hand. That’d be really annoying and unnecessary, so we have a tool for that – `abuild checksum`. Have you read [guide](https://wiki.alpinelinux.org/wiki/Creating_an_Alpine_package#APKBUILD_variables.2Ffunctions) on the Alpine wiki?